pkg/pool-utils/contracts/BasePool.sol

```diff
@@ -178,11 +178,11 @@ abstract contract BasePool is IBasePool, BasePoolAuthorization, BalancerPoolToke
         emit SwapFeePercentageChanged(swapFeePercentage);
     }
 
-    function _getMinSwapFeePercentage() internal pure virtual returns (uint256) {
+    function _getMinSwapFeePercentage() internal view virtual returns (uint256) {
```

Comment:
I need to use view here because of inherited class SwapFeeControllableBasePool uses read operations.
```
    function _getMinSwapFeePercentage() internal view virtual override returns (uint256) {
        if (_currentSwapFeePercentage != 0) {
            return _currentSwapFeePercentage;
        }
        return _DEFAULT_SWAP_FEE_PERCENTAGE;
    }
```